vignettes/cancensus.Rmd

```diff
@@ -0,0 +1,195 @@
+---
+title: "cancensus"
+author: 
+- name: "Dmitry Shkolnik"
+date: "11/20/2017"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, echo=FALSE, message=FALSE, warning=FALSE}
+knitr::opts_chunk$set(include = TRUE)
+library(cancensus)
+library(dplyr)
+# options(cancensus.api_key = "your_api_key")
+```
+# Canadian Census Data
```

Comment:
I would kill this section. It's basically just complaining about StatsCan, and it isn't necessary. The next section speaks to the merits of the package well enough in its own right.

Reply:
Sure, I agree. There's a sore spot with STC and their lack of activity on this front, and I can't deny that it wasn't a motivation behind all of this, but you're right.

Reply:
Yeah, I totally understand the sentiment.